<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>461</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
    </property>
    <property name="title">
     <string>Select Location</string>
    </property>
    <widget class="QComboBox" name="stateList">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);

font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>State</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>City</string>
     </property>
    </widget>
    <widget class="QListWidget" name="cityList">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QListWidget" name="zipList">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Zipcode</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>130</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>781</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
    </property>
    <property name="title">
     <string>Filter on Categories</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Select Category</string>
     </property>
    </widget>
    <widget class="QListWidget" name="categoryList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Businesses</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="businessTable">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>591</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Lucida Grande&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
    </property>
    <property name="title">
     <string>Zipcode Statistics</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string># of Businesses:</string>
     </property>
    </widget>
    <widget class="QLabel" name="numBusiness">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="totalPop">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="avgIncome">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Population:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Average Income:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>781</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableWidget" name="popularTable">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>351</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Lucida Grande&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QTableWidget" name="successfulTable">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>361</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Lucida Grande&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Popular Businesses (In Zipcode)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Successful Businesses (In Zipcode)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="busButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBusiness_Page">
    <property name="title">
     <string>Business Page:</string>
    </property>
   </widget>
   <addaction name="menuBusiness_Page"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
